app/models/samples/json/schedule_ruleset_office_occupancy.json

```diff
@@ -118,7 +118,7 @@
             "type": "ScheduleRule",
             "schedule_day": {
                 "type": "ScheduleDay",
-                "name": "Large Office Bldg Occ_WntrDsnSun",
```

Comment:
@chriswmackey , this example file does not include the new changes to scheduleruleset as well